coderd/database/queries/provisionerdaemons.sql

```diff
@@ -44,7 +44,10 @@ SELECT
 	current_job.id AS current_job_id,
 	current_job.job_status AS current_job_status,
 	previous_job.id AS previous_job_id,
-	previous_job.job_status AS previous_job_status
+	previous_job.job_status AS previous_job_status,
+	tmpl.name AS template_name,
+	tmpl.display_name AS template_display_name,
+	tmpl.icon AS template_icon
```

Comment:
These should be named "current_job_...". Do we also need to include the workspace information?

Reply:
For now, in the design, I only see us using the template data. We can add workspace info later on if necessary.